### DISCUSSION TITLE
Domain

### DISCUSSION THREAD

**Discussion by legacyxdd** [vercel/community#2487] [Help]
Hello, I had a domain with the name "legacy", I deleted that site and created a new one with the name "legacy" again, but it says that name is taken, when I go to "legacy.vercel.app" nothing pops up, I want to claim that name again, what can I do?

**Comment by smaeda-ks** [comment]
@yigitlegacy Unfortunately, since it is a shared domain, if it's already held by another account, there's not much we can do. Please consider using different aliases or using your own custom domain.

**Comment by smaeda-ks** [comment]
@yigitlegacy while I understand your frustration, please be respectful to others. This is a public community discussion.
https://github.com/vercel/community/blob/main/.github/CODE_OF_CONDUCT.md

Correct me if I'm wrong, but you're trying to add `legacy.vercel.app` to your project, right? I already checked our system and confirmed it's indeed already held by another Vercel account. What likely happened is, the current owner of `legacy.vercel.app` had been in the exact same state as what you're seeing right now. But since you released this domain, the ownership immediately moved to them, if that makes sense.
